pandas/io/parsers.py

```diff
@@ -2194,16 +2194,17 @@ def _handle_usecols(self, columns, usecols_key):
         usecols_key is used if there are string usecols.
         """
         if self.usecols is not None:
-            if any([isinstance(u, string_types) for u in self.usecols]):
+            if any([isinstance(usecol, string_types)
+                    for usecol in self.usecols]):
```

Comment:
ah ok, could also use `c` or `col` 

It's not a 'problem', but IMO a multiline list comprehension or multiline if condition is always less clear than a single line one (if you are not too brief to get it on one line).